packages/backend/vite.config.js

```diff
@@ -47,12 +47,9 @@ const config = {
       formats: ['cjs'],
     },
     rollupOptions: {
-      external: [
-        '@podman-desktop/api',
-        ...builtinModules.flatMap(p => [p, `node:${p}`]),
-      ],
+      external: ['@podman-desktop/api', ...builtinModules.flatMap(p => [p, `node:${p}`])],
       output: {
-        entryFileNames: '[name].js',
+        entryFileNames: '[name].cjs',
```

Comment:
question: I don't know how it can work in your app if you change the extension suffix but not changing the path in package.json ?

Reply:
Replaced commit with one to only add the missing extension.